### PR TITLE
add workingLayer to pm:drawstart

### PR DESCRIPTION
**Pull request by richorama**
addresses #218

### CODE REVIEW

**Comment by richorama** [comment]
@codeofsumit I  have updated the docs and demo

File: README.md [hunk 1]
```diff
@@ -121,17 +121,24 @@ map.pm.enableDraw('Circle', options);
 map.pm.Draw.getShapes();
 
 // listen to when drawing mode gets enabled
-map.on('pm:drawstart', function(e) {//...});
-
+map.on('pm:drawstart', function(e) {
+    e.shape; // the name of the shape being drawn (i.e. 'Circle')
+    e.workingLayer; // the leaflet layer displayed while editing
```
Comment (codeofsumit) [inline README.md]: change the doc sentence here to 
`the leaflet layer displayed while drawing`
and we're good to merge 👍 😄

**Comment by codeofsumit** [comment]
checked it - just a minor word change - once this is done I'll merge. Thanks a ton for your PR! Always nice to get new contributors!

**Comment by codeofsumit** [comment]
nevermind I merge it now and change the word myself - need it for the next feature :-)
Great job!

**Comment by richorama** [comment]
Thank you!

**Pull request merged**